README.md

```diff
@@ -0,0 +1,74 @@
+# Helm - The Kubernetes Package Manager
+
+Helm helps you bootstrap your Kubernetes cluster with **Charts** that provide ready-to-use workloads like:
+
+- A Postgres database
+- A scale-out etcd cluster
+- An HAProxy Edge Load Balancer
+
+A Chart is a unit of Kubernetes manifests that reflect best practices as determined by the Helm community.  Helm's [architecture](docs/architecture.md) is heavily influenced by [Homebrew](https://github.com/Homebrew/homebrew).
+
+## Install Helm
+
+```console
+$ go get github.com/deis/helm/...
+```
+
+## Using Helm
+
+To quickly install a standalone etcd instance:
+
+```
+$ helm update
+---> Fetching updates...
+---> Done
+$ helm search etcd
+	etcd-standalone: etcd-standalone is a single Etcd instance (no cluster)
+	etcd-ha-cluster: an HA Etcd cluster (3+ pods)
+	etcd-discovery:  a single-pod Etcd service discovery server
+$ helm info etcd-standalone
+	Description: etcd-standalone is a single Etcd instance (no cluster)
+	Version: 2.2.0-beta3
+	Website: https://github.com/coreos/etcd
+	Built: Oct. 21, 2015
+	Provides: etcd-rc.yaml etcd-service.yaml
+$ helm install etcd-standalone
```

Comment:
Do we care about mentioning that people need:

- kubectl installed + in their path?
- a kubernetes cluster that the kubectl command can talk to?